crowbar_framework/app/controllers/installer/installers_controller.rb

```diff
@@ -0,0 +1,106 @@
+#
+# Copyright 2015, SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module Installer
+  class InstallersController < ApplicationController
+    skip_before_filter :enforce_installer
+    before_filter :hide_navigation
+
+    def show
+      @steps = Crowbar::Installer.steps
+    end
+
+    #
+    # Perform Crowbar Installation
+    #
+    # Provides the restful api call for
+    # /installer/status 	GET 	return done steps, error and success
```

Comment:
we have to either get rid of, or update the routes in the comments. i would prefer update